java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -672,4 +673,79 @@ CompletableFuture<Object[][]> xpending(
             StreamRange end,
             long count,
             StreamPendingOptions options);
+
+    /**
+     * Changes the ownership of a pending message.
+     *
+     * @see <a href="https://valkey.io/commands/xclaim/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param group The consumer group name.
+     * @param consumer The group consumer.
+     * @param minIdleTime The minimum idle time for the message to be claimed.
+     * @param ids A array of entry ids.
+     * @return A <code>Map</code> of message entries with the format <code>
+     *     {"entryId": ["entry", "data"], ...}</code> that are claimed by the consumer.
```

Comment:
Oh wait it is a 2D array like we have in `xread`:
```
127.0.0.1:6379> XCLAIM mystream mygroup Alice 3600000 1719710679916-0 1719710688676-0
1) 1) "1719710679916-0"
   2) 1) "foo"
      2) "bar"
      3) "foo"
      4) "bar2"
      5) "some"
      6) "value"
2) 1) "1719710688676-0"
   2) 1) "foo"
      2) "bar2"
```
So it should have the same return type with 2D array (array of pairs)

Reply:
Probably you can copy doc from `xread` then and/or update both